Automattic/newspack-plugin
# docs: add documentation for WP CLI commands

### All Submissions:






### Changes proposed in this Pull Request:



Adds missing documentation for custom WP CLI commands registered by this plugin and standardizes documentation for all commands into the callback methods' docblocks for use with the global `--help` flag. 

### How to test the changes in this Pull Request:

Test each of the commands with the `--help` flag and confirm that the output accurately describes the script's purpose and available params, e.g.

<img width="1505" alt="Screenshot 2024-11-11 at 4 00 16 PM" src="https://github.com/user-attachments/assets/f545b0a9-6da9-46a1-a11b-20f9c3d5fa51">

Commands covered: 

- `wp newspack ras setup`
- `wp newspack verify-reader`
- `wp newspack migrate-co-authors-guest-authors`
- `wp newspack backfill-non-editing-contributors`
- `wp newspack schedule-co-authors-author-term-backfill`
- `wp newspack esp sync`

### Other information:
